blogs.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Page Title</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="mystyle.css">
+</head>
+<body>
+
+<div class="header">
+  <h1>My Website</h1>
+  <img src="./images/adoptadog-logo.svg" alt="Logo" width="1500" height="800">
```

Comment:
For now those width and height are OK, but make sure to instead handle the styling in the css